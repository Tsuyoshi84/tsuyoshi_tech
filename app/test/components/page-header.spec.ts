import { mount, RouterLinkStub } from '@vue/test-utils'
import PageHeader from '@/components/page-header.vue'

const makeWrapper = () => {
  return mount(PageHeader, {
    stubs: {
      NuxtLink: RouterLinkStub
    }
  })
}

describe('PageHeader', () => {
  test('is a Vue instance', () => {
    const wrapper = makeWrapper()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('generate DOM', () => {
    const wrapper = makeWrapper()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
