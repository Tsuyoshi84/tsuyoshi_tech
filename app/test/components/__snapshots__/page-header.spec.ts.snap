// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader generate DOM 1`] = `<header></header>`;
